linux_os/guide/system/network/network-nftables/set_nftables_loopback_traffic/ansible/shared.yml

```diff
@@ -0,0 +1,38 @@
+# platform = multi_platform_sle
+
+- name: Implement loopback rules
+  command: nft add rule inet filter input iif lo accept
```

Comment:
Please, use fully-qualified collection names [(FQCN)](https://ansible-lint.readthedocs.io/rules/fqcn/)
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible

Also valid for other tasks in this file.